recipes/poco/all/conanfile.py

```diff
@@ -42,8 +42,8 @@ class PocoConan(ConanFile):
         "PocoJWT": _PocoComponent("enable_jwt", True, ("PocoJSON", "PocoCrypto", ), True),
         "PocoMongoDB": _PocoComponent("enable_mongodb", True, ("PocoNet", ), True),
         "PocoNet": _PocoComponent("enable_net", True, ("PocoFoundation", ), True),
-        "PocoNetSSL": _PocoComponent("enable_netssl", True, ("PocoCrypto", "PocoUtil", "PocoNet", ), True),    # also external openssl
-        "PocoNetSSLWin": _PocoComponent("enable_netssl_win", True, ("PocoNet", "PocoUtil", ), True),
+        "PocoNetSSL": _PocoComponent("enable_netssl", not tools.os_info.is_windows, ("PocoCrypto", "PocoUtil", "PocoNet", ), True),    # also external openssl
+        "PocoNetSSLWin": _PocoComponent("enable_netssl_win", tools.os_info.is_windows, ("PocoNet", "PocoUtil", ), True),
```

Comment:
should it be `self.settings.os == Windows` instead? my doubt that it may break cross-building scenarios (e.g. Windows -> Linux or Linux -> Windows)

Reply:
How can I use self in class namespace?

Reply:
@SSE4 could you elaborate on this, please?

Reply:
I don't think it's possible. probably, you need to add both components and then process it in `config_options` or somewhere else.
the issue is running on Linux with `self.settings.os == Windows`, `tools.os_info.is_windows` will return False.

Reply:
I would try replacing tools.os_info.is_windows with self.settings.os == Windows as suggested by @SSE4.

Reply:
I think we can leave this option to True, since it is removed in other OSs different from windows